homeassistant/components/particle/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""Integration with the Particle Cloud."""
+
+DOMAIN = 'particle'
+
+import logging
+import voluptuous as vol
+from homeassistant.const import (
+  STATE_ON,
+  STATE_UNAVAILABLE,
+)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.config_validation import ENTITY_SERVICE_SCHEMA
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_component import EntityComponent
+
+_LOGGER = logging.getLogger(DOMAIN)
+CONFIG_SCHEMA = vol.Schema({
+  DOMAIN: vol.Schema({
+    vol.Required('access_token'): cv.string,
+  }),
+}, extra=vol.ALLOW_EXTRA)
+
+FUNCTION_CALL_SCHEMA = ENTITY_SERVICE_SCHEMA.extend({
+  vol.Required('function'): cv.string,
```

Comment:
Use a constant here.

```suggestion
  vol.Optional(ATTR_FUNCTION): vol.All(
```

along with `ATTR_FUNCTION = 'function'`